<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swiss Army Knife Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_bin">
      <property name="text">
       <string>Binary</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_bin"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_dec">
      <property name="text">
       <string>Decimal</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_dec"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_oct">
      <property name="text">
       <string>Octal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_oct"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_hex">
      <property name="text">
       <string>Hex</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_hex"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuConversion">
     <property name="title">
      <string>Conversion</string>
     </property>
     <widget class="QMenu" name="menuMath">
      <property name="title">
       <string>Maths</string>
      </property>
      <addaction name="actionBinary_Decimal_Octal_Hexadecimal"/>
     </widget>
     <widget class="QMenu" name="menuPhysics">
      <property name="title">
       <string>Physics</string>
      </property>
      <addaction name="actionCelsius_Fahrenheit_Kelvin"/>
     </widget>
     <widget class="QMenu" name="menuCurrency">
      <property name="title">
       <string>Currency</string>
      </property>
      <addaction name="actionCurrent_exchange_rates"/>
      <addaction name="separator"/>
      <addaction name="actionExchange_rate_trend"/>
     </widget>
     <addaction name="menuMath"/>
     <addaction name="menuPhysics"/>
     <addaction name="menuCurrency"/>
    </widget>
    <addaction name="menuConversion"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBinary_Decimal_Octal_Hexadecimal">
   <property name="text">
    <string>Binary / Decimal / Octal / Hexadecimal</string>
   </property>
  </action>
  <action name="actionCelsius_Fahrenheit_Kelvin">
   <property name="text">
    <string>Celsius / Fahrenheit / Kelvin</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCurrent_exchange_rates">
   <property name="text">
    <string>Current exchange rates</string>
   </property>
  </action>
  <action name="actionExchange_rate_trend">
   <property name="text">
    <string>Exchange rate trend</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
